### PR TITLE
neofs-cli: check if container contains locks before removal

### LINKED ISSUE #1500: neofs-cli: Check if the container has lock objects before the deletion
`neofs-cli` could fail to remove container if there are any active locks in it.
This check should be disabled with some flag like `--force` or `--yes-i-know-what-i-am-doing`.

### PR DESCRIPTION
**Pull request by fyrchik**
Close #1500

### CODE REVIEW

**Comment by fyrchik** [comment]
Ready for review, but let's merge it after release.

File: cmd/neofs-cli/modules/container/delete.go [hunk 1]
```diff
@@ -30,6 +35,26 @@ Only owner of the container has a permission to remove container.`,
...
+			searchPrm.SetContainerID(id)
+			searchPrm.SetFilters(fs)
+			searchPrm.SetTTL(2)
+
+			common.PrintVerbose("Searching for LOCK objects...")
+			res, err := internalclient.SearchObjects(searchPrm)
+			common.ExitOnErr(cmd, "can't search for LOCK objects: %w", err)
+
+			if len(res.IDList()) != 0 {
+				common.ExitOnErr(cmd, "", errors.New("Container wasn't removed because LOCK objects were found.\n"+
```
Comment (carpawell) [inline cmd/neofs-cli/modules/container/delete.go:53]: why not `fmt.Errorf`?

**Review by carpawell — DISMISSED** [review]
Not sure if we need to add expiration to the lock objects in that PR if the logic gonna be changed anyway: https://github.com/nspcc-dev/neofs-node/commit/9100b0ded922ec3197be90b1152a9b3da586524b

File: cmd/neofs-cli/modules/container/delete.go (hunk 1 above)
Comment (fyrchik) [reply cmd/neofs-cli/modules/container/delete.go]: Fixed

**Comment by fyrchik** [comment]
@carpawell we could just disallow zero values when merging that branch.

**Pull request merged**